src/test/java/org/fundacionjala/coding/franz/BankOCRTest.java

```diff
@@ -0,0 +1,135 @@
+package org.fundacionjala.coding.franz;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+/**
+ * this is a test of class.
+ */
+public class BankOCRTest {
+    private BankOCR bankNumber;
+
+    /**
+     * this is a method before test.
+     */
+    @Before
+    public void ini() {
+        bankNumber = new BankOCR();
+    }
+
+    /**
+     * this is a test of method accountNumber().
+     */
+    @Test
+    public void tesyAccountNumbersNine() {
+        String lineOne = " _  _  _  _  _  _  _  _  _ ";
+        String lineTwo = "|_||_||_||_||_||_||_||_||_|";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define a constant instead of duplicating this literal "|_||_||_||_||_||_||_||_||_|" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)